src/app/pages/CpsSty/index.jsx

```diff
@@ -6,10 +6,11 @@ import {
   GEL_SPACING_TRPL,
   GEL_SPACING_QUAD,
 } from '@bbc/gel-foundations/spacings';
+import { C_CHALK } from '@bbc/psammead-styles/colours';
```

Comment:
This may need confirming with UX. I just picked the closest available colour to that of the sidebar on the PAL.

Reply:
Yes, lets check that passes for colour contrast...